### PR TITLE
Reproduce Caffe results

### PR DESCRIPTION
**Pull request by hgaiser**
This PR includes the changes from https://github.com/broadinstitute/keras-resnet/pull/34. In addition, this PR:

- Disables bias for convolutional layers, as per the original implementation.
- Configures `BatchNormalization` to use `epsilon=1e-5`, as per Caffe's default.
- Changes some `padding="same"` to `ZeroPadding2D`. The difference is that `ZeroPadding2D` pads symmetrically (same as `padding=3` in Caffe) whereas `padding="same"` pads at the end of a blob until it can maintain the correct shape.
- Adds tools for importing / exporting ResNet models from Caffe (first convert Caffe model to `.h5` file, then import into Keras).

With these changes I got near identical (within reasonable floating point precision) results compared to Caffe output for a few test images.

Exported models from Caffe can be downloaded [here](https://delftrobotics-my.sharepoint.com/personal/h_gaiser_fizyr_com/_layouts/15/guestaccess.aspx?folderid=0348cbc8beb5748d9a0805eebd9b68c09&authkey=AYEWXlUd8yH6uukiEvVKRPo&e=f7a4029f4f2e4352bb0c2f9a13bc8379).

### CODE REVIEW

**Comment by hgaiser** [comment]
I added a commit to rename `BatchNormalization` to `FreezeBatchNormalization` as the original name was giving conflicts with the name from Keras (when trying to do a `load_model` on a `keras-resnet` model it was trying to load `keras.layers.BatchProcessing` instead of `keras_resnet.layers.BatchProcessing`).

**Comment by hgaiser** [comment]
I added the models on github: https://github.com/fizyr/keras-models/releases/tag/v0.0.1

**Comment by 0x00b1** [comment]
Fantastic! Thanks, @hgaiser.
